src/vs/workbench/contrib/notebook/browser/notebookExtensionPoint.ts

```diff
@@ -42,6 +42,16 @@ export interface INotebookRendererContribution {
 	readonly [NotebookRendererContribution.requiresMessaging]: RendererMessagingSpec;
 }
 
+const NotebookPreloadContribution = Object.freeze({
+	type: 'type',
+	entrypoint: 'entrypoint',
+});
+
+export interface INotebookPreloadContribution {
+	readonly [NotebookPreloadContribution.type]: string;
+	readonly [NotebookPreloadContribution.entrypoint]: string;
```

Comment:
Will these scripts have access to the messaging channel, i.e. can the webview scripts send messages back to the exension host?
From what I can tell they can, however I'm not sure how the extension can listen to these messages, as we don't have an identifier to setup a comms channel.

Note: This was never a requirement, just curious about the capabilities.